saleor/graphql/product/types.py

```diff
@@ -42,6 +42,16 @@ class StockAvailability(graphene.Enum):
     OUT_OF_STOCK = 'OUT_OF_STOCK'
 
 
+class ProductOrderField(graphene.Enum):
+    NAME = 'name'
+    PRICE = 'price'
+
+
+class OrderDirection(graphene.Enum):
```

Comment:
Let's add descriptions for these values as well:
- ASC - "Specifies an ascending sort order."
- DESC - "Specifies a descending sort order."